source/grid/tria.cc

```diff
@@ -1689,7 +1689,7 @@ namespace internal
               default:
                 // in 1d, a node must have one or two adjacent lines
                 if (spacedim==1)
-                  AssertThrow (false, ExcInternalError())
+                  Assert (false, ExcInternalError())
```

Comment:
This line is completely bizarre. How is it possible that we have an assertion without a semicolon? I think it would be best to just stick with the general exception here.